docs/ops/movement/GatherElements_6.md

```diff
@@ -0,0 +1,126 @@
+## GatherElements <a name="GatherElements"></a> {#openvino_docs_ops_movement_GatherElements_6}
+
+**Versioned name**: *GatherElements-6*
+
+**Category**: Data movement operations
+
+**Short description**: *GatherElements* takes elements from input `data` tensor at positions specified in `indices`.
```

Comment:
Please, add "the" before all tensor names like: `data`, `indices` etc.

Reply:
Done.